web/modules/custom/customhook/customhook.module

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * @file
+ * This file demonstrates  how to use hook function
+ */
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\Core\Entity\Entity\EntityViewDisplay;
+
+/**
+ * Implements hook_cron()
+ */
+function customhook_cron() {
+  \Drupal::messenger()->addMessage(t('This message is from cron hook'));
+}
+
+/**
+ * Implements hook_form_alter()
+ */
+function customhook_form_alter(&$form, \Drupal\Core\Form\FormStateInterface $form_state, $form_id) {
```

Comment:
Check `book_form_node_confirm_form_alter` in core/ how this particular line is written.